app/ante/cosmos/sigverify.go

```diff
@@ -257,7 +259,7 @@ func OnlyLegacyAminoSigners(sigData signing.SignatureData) bool {
 }
 
 func (svd SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
-	if isOracleCreatePriceTx(tx) {
+	if utils.IsOracleCreatePriceTx(tx) {
 		// TODO: verify ed25519 signature for create-price message which is signed by consensusKey
```

Comment:
Can the price transaction now be submitted by any account? If yes, it seems like this check should be implemented before mainnet.

Reply:
> Can the price transaction now be submitted by any account? If yes, it seems like this check should be implemented before mainnet.

yes, in this PR it is ensured that tx singed by validator's consensuskey will be accept only